user-integration-guide/symfony-integration.md

```diff
@@ -0,0 +1,218 @@
+# Integrating Opentelemetry PHP into Symfony Applications
+
+## Introduction 
+
+As a developer, you might be wondering how  OpenTelemetry could be beneficial to you. You are not alone.  Without practical examples, the usefulness of distributed tracing can be difficult to grasp for persons without a cloud or site reliability engineering background. This user guide shows how OpenTelemetry could be useful to gain insights into exceptions happening within an application. This example uses the OpenTelemtry PHP library integrated into a Symfony application, bundled with Jaeger and Zipkin, for visualizing data.  
+## Prerequisites
+To follow this guide you will need:
+
+* PHP Installed, this example uses PHP 7.4 .
+* [Composer](https://getcomposer.org/download/ ) for dependency management. 
+* [Symfony CLI](https://symfony.com/download) for managing your Symfony application
+*  [Docker](https://docs.docker.com/get-docker/) for bundling our visualization tools
+
+## Step 1 - Creating a Symfony Application 
+
+Create a Symfony application by running the command `symfony new my_project_name`. Remember that you need the  [Symfony CLI](https://symfony.com/download) installed for the `symfony` command to work. I am calling this example `otel-php-symfony-basic-example`, so the command is as follows; `symfony new otel-php-symfony-basic-example` .
```

Comment:
It's often helpful to have a command the user needs to copy & paste into their terminal live on its own line.  That way it's easy to copy and paste.

Reply:
You also probably don't need to reference the `Symfony CLI` here, because you already mentioned it in the prerequisite part of this document.